CHANGES.rst

```diff
@@ -37,6 +37,9 @@ Changes
 - The `packaging` dependency was removed.
   :pr:`1307` by :user:`Jovan Stojanovic <jovan-stojanovic>`
 
+- :class:`TextEncoder`, :class:`StringEncoder` and :class:`GapEncoder` outputs during
+  transform are now normalized by computing the total standard deviation during
```

Comment:
maybe be a bit more verbose to make it clearer that the whole thing is rescaled by a global constant, and this is not row or column normalization